python/monitoring/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM mesosphere/python-base:latest
+MAINTAINER Sunil Shah <sunil@mesosphere.io>
```

Comment:
Let's change this to `Mesoshpere, Inc. <support@mesosphere.io>` unless you like reading emails.